examples/c++-api/Makefile

```diff
@@ -0,0 +1,7 @@
+all: count-stuff
+
+clean:
+	-rm count-stuff 
+
+count-stuff: count-stuff.cc
+	$(CXX) -I ../../lib count-stuff.cc -o count-stuff ../../lib/liboxli.a
```

Comment:
Ok, got it.

It doesn't look like any of the current makefile targets build `liboxli.a`. I assume this is related to your recent comments about this not currently being a priority support item.